tests/link/test_numba.py

```diff
@@ -2112,51 +2112,6 @@ def test_Softplus(x, exc):
         )
 
 
-@pytest.mark.parametrize(
-    "x, axes, exc",
-    [
-        (
-            set_test_value(at.dscalar(), np.array(0.0, dtype="float64")),
-            [],
-            None,
-        ),
-        (
-            set_test_value(at.dvector(), rng.random(size=(3,)).astype("float64")),
-            [0],
-            None,
-        ),
-        (
-            set_test_value(at.dmatrix(), rng.random(size=(3, 2)).astype("float64")),
-            [0],
-            None,
-        ),
-        (
-            set_test_value(at.dmatrix(), rng.random(size=(3, 2)).astype("float64")),
-            [0, 1],
-            None,
-        ),
-    ],
-)
-def test_MaxAndArgmax(x, axes, exc):
-    g = aem.MaxAndArgmax(axes)(x)
-
-    if isinstance(g, list):
-        g_fg = FunctionGraph(outputs=g)
-    else:
-        g_fg = FunctionGraph(outputs=[g])
-
-    cm = contextlib.suppress() if exc is None else pytest.warns(exc)
-    with cm:
-        compare_numba_and_py(
-            g_fg,
-            [
-                i.tag.test_value
-                for i in g_fg.inputs
-                if not isinstance(i, (SharedVariable, Constant))
-            ],
-        )
```

Comment:
We can add a new `pytest.mark.parametrize` and parameter that cycles through the two now independent `Op`s and runs the same tests.